big_tests/tests/cets_disco_SUITE.erl

```diff
@@ -63,14 +64,47 @@ file_backend(Config) ->
     ['node1@localhost', 'node2@otherhost'] = lists:sort(Nodes).
 
 rdbms_backend(_Config) ->
-    Opts1 = #{cluster_name => <<"big_test">>, node_name_to_insert => <<"test1">>},
-    Opts2 = #{cluster_name => <<"big_test">>, node_name_to_insert => <<"test2">>},
-    State1 = rpc(mim(), mongoose_cets_discovery_rdbms, init, [Opts1]),
-    rpc(mim(), mongoose_cets_discovery_rdbms, get_nodes, [State1]),
-    State2 = rpc(mim(), mongoose_cets_discovery_rdbms, init, [Opts2]),
-    {{ok, Nodes}, State2_2} = rpc(mim(), mongoose_cets_discovery_rdbms, get_nodes, [State2]),
+    CN = <<"big_test">>,
+    Opts1 = #{cluster_name => CN, node_name_to_insert => <<"test1">>},
+    Opts2 = #{cluster_name => CN, node_name_to_insert => <<"test2">>},
+    State1 = disco_init(Opts1),
+    disco_get_nodes(State1),
+    State2 = disco_init(Opts2),
+    {{ok, Nodes}, State2_2} = disco_get_nodes(State2),
     %% "test2" node can see "test1"
     true = lists:member(test1, Nodes),
-    {{ok, _}, State2_3} = rpc(mim(), mongoose_cets_discovery_rdbms, get_nodes, [State2_2]),
+    {{ok, _}, State2_3} = disco_get_nodes(State2_2),
     %% Check that we follow the right code branch
     #{last_query_info := #{already_registered := true}} = State2_3.
+
+rdbms_backend_supports_auto_cleaning(_Config) ->
+    CN = <<"big_test2">>,
+    Opts1 = #{cluster_name => CN, node_name_to_insert => <<"test1">>, override_timestamp => month_ago()},
```

Comment:
A bit artificial (option introduced only for tests). I'd rather use meck here, but it's my personal preference.

Reply:
yeaa, maybe could use timestamp_module opt :) 
But yea, could rewrite with meck...